### PR TITLE
Stream block summaries EP

### PR DESCRIPTION
**Pull request by semyonoskin**

### CODE REVIEW

File: modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/models/FullBlockInfoV1.scala [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import io.circe.magnolia.derivation.encoder.semiauto.deriveMagnoliaEncoder
+import org.ergoplatform.explorer.db.models.aggregates.{ExtendedAsset, ExtendedDataInput, ExtendedInput, ExtendedOutput}
+import org.ergoplatform.explorer.db.models.{Header, Transaction}
+import org.ergoplatform.explorer.http.api.v0.models.{HeaderInfo, TransactionInfo => TxInf}
+import sttp.tapir.{Schema, Validator}
+
+final case class FullBlockInfoV1(
+                                header: HeaderInfo,
+                                blockTransactions: List[TxInf],
+                              )
```
Comment (oskin1) [inline modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/models/FullBlockInfoV1.scala:14]: Fmt.

File: modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/models/FullBlockInfoV1.scala [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+import io.circe.Codec
+import io.circe.magnolia.derivation.decoder.semiauto.deriveMagnoliaDecoder
+import io.circe.magnolia.derivation.encoder.semiauto.deriveMagnoliaEncoder
+import org.ergoplatform.explorer.db.models.aggregates.{ExtendedAsset, ExtendedDataInput, ExtendedInput, ExtendedOutput}
+import org.ergoplatform.explorer.db.models.{Header, Transaction}
+import org.ergoplatform.explorer.http.api.v0.models.{HeaderInfo, TransactionInfo => TxInf}
+import sttp.tapir.{Schema, Validator}
+
+final case class FullBlockInfoV1(
```
Comment (oskin1) [inline modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/models/FullBlockInfoV1.scala:11]: No need in V1 suffix, the class is already in api.v1 package.

File: modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/models/BlockSummaryV1.scala [hunk 3]
```diff
@@ -0,0 +1,24 @@
+package org.ergoplatform.explorer.http.api.v1.models
+
+import io.circe.Codec
+import io.circe.magnolia.derivation.decoder.semiauto.deriveMagnoliaDecoder
+import io.circe.magnolia.derivation.encoder.semiauto.deriveMagnoliaEncoder
+import org.ergoplatform.explorer.http.api.v0.models.BlockReferencesInfo
+import sttp.tapir.{Schema, Validator}
+
+final case class BlockSummaryV1(block: FullBlockInfoV1, references: BlockReferencesInfo)
```
Comment (oskin1) [inline modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/models/BlockSummaryV1.scala:9]: No need in V1 suffix

File: modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/models/FullBlockInfoV1.scala [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  def apply(
+             h: Header,
+             txs: List[Transaction],
+             numConfirmations: Int,
+             inputs: List[ExtendedInput],
+             dataInputs: List[ExtendedDataInput],
+             outputs: List[ExtendedOutput],
+             assets: List[ExtendedAsset],
+             blockSize: Int
+           ): FullBlockInfoV1 = {
```
Comment (oskin1) [inline modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/models/FullBlockInfoV1.scala:35]: Fmt.

File: modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/models/BlockSummaryV1.scala (hunk 3 above)
Comment (oskin1) [reply modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/models/BlockSummaryV1.scala:9]: Let's keep only FullBlockInfo, no need for BlockReferencesInfo

File: modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/models/FullBlockInfoV1.scala (hunk 2 above)
Comment (semyonoskin) [reply modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/models/FullBlockInfoV1.scala:11]: Models with same names are used in same files. Isn't this handier than import aliases ?

**Review by oskin1 — APPROVED** [review]

**Pull request merged**